lib/api.rb

```diff
@@ -218,7 +235,9 @@ def validation_error(msg)
       if !u.save
         return validation_error("User save failed")
       end
-
+      status 200
+      headers \
+      "access-control-allow-origin" => "*"
```

Comment:
`status 200` shouldn't be needed, and the `headers` line can be put on one line.